ood-portal-generator/lib/ood_portal_generator/dex.rb

```diff
@@ -10,15 +10,12 @@ module OodPortalGenerator
   class Dex
     # @param opts [#to_h] the options describing the context used to render the Dex config
     def initialize(opts = {}, view = nil, insecure = false)
+      opts = {} unless opts.respond_to?(:to_h)
```

Comment:
Nobody has ever hit this bug because the example yaml we share has a least the YAML seperator `---` which apparently is significant.